chain-crypto/src/algorithms/vrf/vrf.rs

```diff
@@ -32,7 +27,7 @@ impl AsRef<[u8]> for SecretKey {
 
 /// VRF Public Key
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct PublicKey(Point, CompressedRistretto);
+pub struct PublicKey(GroupElement, [u8; PUBLIC_SIZE]);
```

Comment:
Why does it have both a point and it's compressed form?

Reply:
I thought the same, that it was redundant. The problem is that if the public key is only a `GroupElement`, then we cannot implement the `AsRef` trait. The other option would be to only have it's compressed form, but then every time we use the pk we need to decompress.

Reply:
Could `PUBLIC_SIZE` be an associated const: `PublicKey::SIZE`?